code/game/objects/items/storage/toolbox.dm

```diff
@@ -20,22 +20,59 @@
 	material_flags = MATERIAL_EFFECTS | MATERIAL_COLOR
 	var/latches = "single_latch"
 	var/has_latches = TRUE
+	var/latch_exempt = FALSE
```

Comment:
Document

Reply:
Do you want me to add documentation for the older vars too, like has_latches, since im using that var in my code?